runtime/tutor

```diff
@@ -339,6 +339,12 @@ _________________________________________________________________
  3. Move to the fourth line.
  4. Type x twice or type 2x to select 2 lines, and d to delete.
 
+ This also works in reverse.
+
+ 1. Move the cursor to line labeled "7".
+ 2. Type X to select the current line.
+ 3. Type X again to select the previous line, and d to delete.
```

Comment:
they are? 1.3 is 13 lines, 1.4 is 17, 1.5 is 19, 2.1 is 20 lines long. If there is a reason for keeping 18 length sections, it's a rule thats already broken.

Reply:
I mean the spacing of the sections rather than the content:

```
=================================================================
=                     X.Y SECTION TITLE                         =
=================================================================
L1
L2
L3
L4
L5
L6
L7
L8
L9
L10
L11
L12
L13
L14
L15
L16
L17
L18
L19
```

looks like it's 19 rather than 18 actually. It looks to me like all sections are the correct length on master currently, no?